### DISCUSSION TITLE
Just updated to latest version 10 and this really bothers me

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#1690] [General]
<img width="592" alt="Screenshot 2023-10-13 at 00 57 37" src="https://github.com/LazyVim/LazyVim/assets/59118341/b3a85d93-8a54-4fe3-8bc2-c7af32afc584">

It tracks the current method or whatever, and it does nothing but making my editor verbose.

How do I disable it?

**Comment by jyuan0** [comment]
Add this spec: ```{ "nvim-treesitter/nvim-treesitter-context", enabled = false }```

If you want to toggle it: `TSContextToggle`
